server/http.go

```diff
@@ -402,7 +410,7 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 
 	post, appErr := p.API.GetPost(postID)
 	if appErr != nil {
-		p.API.LogWarn("Could not get the meeting post by id", "Error", appErr.Error())
+		p.API.LogWarn("Could not get the meeting post by id", "err", appErr.Error())
```

Comment:
Make this similar to above "failed to get the meeting post by id"

Reply:
Although I did it, Is there any reason behind that?